packages/markers/src/browser/problem/problem-marker.ts

```diff
@@ -5,20 +5,16 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import { Marker, MarkerManager } from '../marker-manager';
+import { MarkerManager } from '../marker-manager';
 import { Diagnostic } from "vscode-languageserver-types";
 import { injectable, inject } from 'inversify';
 import { StorageService } from '@theia/core/lib/browser/storage-service';
 import { FileSystemWatcher } from '@theia/filesystem/lib/common';
-
-export const PROBLEM_KIND = 'problem';
-
-export interface ProblemMarker extends Marker<Diagnostic> {
-    kind: 'problem';
-}
+import { IMarker } from '../../common/marker';
+import { IProblemMarker, PROBLEM_KIND } from '../../common/problem';
 
 export namespace ProblemMarker {
```

Comment:
the static part of the interface should be always next to the interface and have the same name

Reply:
@akosyakov 
Can you clarify this, I don't understand what you mean  by the static part !

Reply:
```ts
class A {
    static bar = 1;
    foo: number;
}
```
is the same as:
```ts
// instance part of A
interface A {
    foo: number;
}
// static part of A
namespace A {
    export const bar = 1;
}
```

they go together

Reply:
Done,
 move to the file ...common/problem-marker.ts